Doc/library/decimal.rst

```diff
@@ -374,10 +374,22 @@ Decimal objects
    returns ``Decimal('1.414')``.
 
    If *value* is a :class:`float`, the binary floating point value is losslessly
-   converted to its exact decimal equivalent.  This conversion can often require
-   53 or more digits of precision.  For example, ``Decimal(float('1.1'))``
-   converts to
-   ``Decimal('1.100000000000000088817841970012523233890533447265625')``.
+   converted to its exact decimal equivalent.  This means, for example, that
+   ``Decimal(0.1)`` is not the same as ``Decimal('0.1')``.
+   Since 0.1 is not exactly representable in binary floating point, the
+   value is stored as the nearest representable value which is
+   ``0x1.999999999999ap-4``.
+
+   .. doctest::
+
+      >>> Decimal(0.1)
+      Decimal('0.1000000000000000055511151231257827021181583404541015625')
```

Comment:
```suggestion
      >>> Decimal(0.1) == Decimal('0.1')
      False
      >>> Decimal(0.1)
      Decimal('0.1000000000000000055511151231257827021181583404541015625')
```

I would add this, mostly because on top of proving what was mentioned in the paragraph, the syntax highlight should make it evident that `0.1` and `'0.1'` are different -- I had to re-read it a few times before I noticed the `'...'`.

Reply:
Nice, thanks!